### PR TITLE
Selectable item styles

### PR DESCRIPTION
**Pull request by jtreminio**

### CODE REVIEW

File: src/Builder/CliMenuBuilder.php [hunk 1]
```diff
@@ -396,22 +397,25 @@ public function setMargin(int $margin) : self
     public function setUnselectedMarker(string $marker) : self
```
Comment (AydinHassan) [inline src/Builder/CliMenuBuilder.php]: Let's just drop these methods. It's just a little BC break.

**Comment by AydinHassan** [comment]
Everything else looks good!

File: src/Builder/CliMenuBuilder.php (hunk 1 above)
Comment (jtreminio) [reply src/Builder/CliMenuBuilder.php]: Still needed for MenuMenuItem. I can completely drop in next PR for that item's style.

Comment (AydinHassan) [reply src/Builder/CliMenuBuilder.php]: Ok great, sounds good.

**Pull request merged**